test/fixtures/dialects/redshift/redshift_create_external_table_as.sql

```diff
@@ -0,0 +1,51 @@
+CREATE EXTERNAL TABLE
+external_schema.table_name
+STORED AS PARQUET
+LOCATION 's3://bucket/folder/'
+AS
+SELECT col1, col2 FROM external_schema.source_table
+;
```

Comment:
in the grammar you mention that this can be optionally bracketed, so in one of the examples could you make this:
```
AS (
    SELECT col1, col2 FROM external_schema.source_table
)
;
```